EMS/core-bundle/src/Repository/TemplateRepository.php

```diff
@@ -76,17 +76,14 @@ public function create(Template $template): void
     }
 
     /**
-     * @param string[] $ids
-     *
      * @return Template[]
      */
-    public function getByIds(array $ids): array
+    public function getByIds(string ...$ids): array
```

Comment:
Good idea, I like it a lot

Reply:
Yes if an array contains only one type is clean to just use unpacking. Removes stupid doc blocks 😄